### PR TITLE
Tapered sections

### LINKED ISSUE #115: Pull Tapered sections
#### Description:
add functionality to read tapered sections from RFEM

### LINKED ISSUE #116: Pull fails when using coupling type
#### Description:
RFEM has several versions of what BHoM calls 'Rigid link' including 'coupling' when coupling is present as a member type Pulling the RFEM model will fail.

#### Steps to reproduce:
assign any 'coupling' type as a member type and try to pull bars.

#### Expected behaviour:
when pulling bars only bars should be pulled, when pulling rigid members this should include all 'coupling' types

### PR DESCRIPTION
**Pull request by MartinHenriksen**
### Issues addressed by this PR

 Closes #
closes #115 
closes #116 

 
added feature to read tapered sections from RFEM and fixed bug when reading rigidlinks with coupling type set. 

 ### Test files

[test file](https://burohappold.sharepoint.com/:u:/r/sites/BHoM/02_Current/12_Scripts/02_Pull%20Request/BHoM/RFEM_Toolkit/RFEM_Toolkit-TaperedSections.gh?csf=1&web=1&e=aHHTK5)
RFEM file not shared here due to confidentiality

 ### Changelog

added feature: read tapered sections
fixed bug: rigid link with type coupling

### CODE REVIEW

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required

**Comment by FraserGreenroyd** [comment]
@BHoMBot check core

**Comment by FraserGreenroyd** [comment]
@BHoMBot check copyright-compliance

**Review by staintono — DISMISSED** [review]
looks good, doesnt throw any red error messages anymore

**Comment by MartinHenriksen** [comment]
made small changes to satisfy code-compliance check. @staintono please re-submit your review

**Review by staintono — DISMISSED** [review]
lgtm

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required

File: RFEM_Engine/Compute/Profile.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 namespace BH.Engine.Adapters.RFEM
 {
-    public static partial class Query
+    public static partial class Compute
```
Comment (FraserGreenroyd) [inline RFEM_Engine/Compute/Profile.cs:38]: Sorry @MartinHenriksen - this change needs to be versioned. Easiest way is with a `PreviousVersion` attribute I suspect! Gimme a shout if you need any help, happy to support :smile:

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
See comment. As comment relates only to compliance, I am happy to approve once checks go green providing only commits are to versioning and not functionality to save you finding a re-review @MartinHenriksen :smile:

File: RFEM_Engine/Compute/Profile.cs (hunk 1 above)
Comment (MartinHenriksen) [reply RFEM_Engine/Compute/Profile.cs:38]: Thanks @FraserGreenroyd I have added the attribute

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check serialisation

**Comment by FraserGreenroyd** [comment]
@BHoMBot check core

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a CI/CD instruction. I am authorising dispensation to be granted on check ref. 2360466119

**Review by FraserGreenroyd — APPROVED** [review]
Approving on a versioning check and using functionality review from previously as discussed with @MartinHenriksen as last commit is only a versioning fix :smile:

**Pull request merged**